wallet/models.go

```diff
@@ -98,31 +100,37 @@ type GetAddressIndexRequest struct {
 // Index model
 type Index struct {
 	// Account index.
-	Major uint64 `json:"major"`
-	// Address index.
-	Minor uint64 `json:"minor"`
+	Major uint32 `json:"major"`
+	// Subaddress index.
+	Minor uint32 `json:"minor"`
```

Comment:
I did [a PR](https://github.com/MarinX/monerorpc/pull/18) on MarinX's repo to fix some features we don't use for the atomic swaps and adjusted these to match what `monero-wallet-rpc` is using to represent them on the server side.